force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -662,14 +662,6 @@
         <shortDescription>Navigation title for Account Model</shortDescription>
         <value>Account Model</value>
     </labels>
-    <labels>
```

Comment:
Looks like there are tons of label changes here mostly repositionings. Just want to make sure if they are on purpose.

Reply:
They are quite deliberate. We fell out of line with the default ordering in the labels file, so this is restoring that order.